react/components/InfoCard/index.js

```diff
@@ -290,6 +298,8 @@ MemoizedInfoCard.defaultProps = {
   textMode: textModeTypes.TEXT_MODE_HTML.value,
   linkTarget: '_self',
   callToActionLinkTarget: '_self',
+  fetchpriority: 'auto',
+  preload: '',
```

Comment:
Since this is a boolean, this prop should be false and not line in isFullModeStyle ''